tests/Rector/MagicDisclosure/GetAndSetToMethodCallRector/Fixture/fixture3.php.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+
+class Enlight_View_Default
+{
+    protected $engine;
+
+    public function __construct($engine)
+    {
+        $this->engine = $engine;
+    }
+}
+?>
+-----
+<?php
+
+class Enlight_View_Default
+{
+    protected $engine;
+
+    public function __construct($engine)
+    {
+        $this->engine = $engine;
+    }
+}
+?>
```

Comment:
If the code should remain the same after Rector run, just keep the first part. 

Remove the code after `-----`